common/changes/office-ui-fabric-react/icon-onClicked-deprecate_2017-09-20-17-13.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Icon: Deprecate onClicked as onClick is already supported. Updated DatePicker to use onClick",
+      "type": "patch"
```

Comment:
is `patch` still okay?

Reply:
I think so. He's not removing it, or adding features.

Reply:
hmmm. This isn't a deprecate... this is a removal.

Reply:
I will minor it.